### ISSUE TITLE
Crashes when trying to prepend multiple items

### ISSUE THREAD

**Issue by christopherdro** [nolimits4web/Swiper#822]
Hi there,

First off I think this script is great and has so many incredible features.

I'm trying to use the swiper to make a carousel of dates from a calendar. I don't want to load all the days of the year at once so I'm trying to load an initial amount (2 weeks) then prepend or append as necessary.

The append works just fine it's just the prepend seems to crash when trying to add multiple items.

The other issue is the sliding to index 0 once an item is prepended.

I saw another issue where you recommended to use swipeTo after the the prepend which works but only once or twice before it crashes. 

For now I had to create my own conditional statements to detect whether the user has reached the beginning or the end of the slider since I didn't see this feature implemented in the documents. If this feature does indeed exist please let me know.

I'm wondering if since prepending requires much more processing since your essential changing the indexes of all slides that come after the prepended one.

Is there a possible way to chain prepends and/or it my case set the start index to something other than 0, i.e. '999' so when I prepend items it can just decrement from the first slide rather then having to change all the indexes.

I hope this make sense and hope you can help. 
Here is a link to test code I have been working on. 

https://gist.github.com/christopherdro/d4189b5f11df53684c1d

Thank you.
